### PR TITLE
Executor calls, GelDataDelivery fix

### LINKED ISSUE #69: gel_upload crash when state is not md5 or qc
https://github.com/EdinburghGenomics/EGCG-Project-Management/blob/e4c6d199cd9e5bc13b6c4ed13c2fe3ca60ed4538/upload_to_gel/deliver_data_to_gel.py#L259
When the state is 'delivered' (meaning it has not been checked yet) this line makes the script crash.

### LINKED ISSUE #100: EGCG-Core#99 compatibility
Project-Management version of EdinburghGenomics/Analysis-Driver#379, except for:

- data_deletion/\_\_init\_\_.py/delete_dir

### PR DESCRIPTION
**Pull request by mwhamgenomics**
- Ensures all cluster_execute calls declare `cpus` and `mem` - closes #100 
- Adds support for status 'delivered' to GelDataDelivery.check_delivery_data - fixes #69

### CODE REVIEW

File: upload_to_gel/deliver_data_to_gel.py [hunk 1]
```diff
@@ -253,26 +253,33 @@ def check_delivery_data(self):
             self.delivery_id,
             'upload_state', 'md5_state', 'md5_confirm_date', 'qc_state', 'qc_confirm_date'
         )
+        msg = ''
+        emit = self.error
```
Comment (tcezard) [inline upload_to_gel/deliver_data_to_gel.py]: That seems unnecessary to replace `self.error` with another variable name

File: upload_to_gel/deliver_data_to_gel.py [hunk 2]
```diff
@@ -253,26 +253,33 @@ def check_delivery_data(self):
...
-            self.error('Delivery %s sample %s: Upload failed', self.delivery_id, self.sample_id)
+            msg = 'Upload failed'
         elif qc_state:
-            self.error('Delivery %s sample %s qc check failed - was checked before on %s', self.delivery_id, self.sample_id, qc_confirm_date)
+            msg = 'QC check failed - was checked before on %s' % qc_confirm_date
         elif md5_state:
-            self.error('Delivery %s sample %s md5 check failed - was checked before on %s', self.delivery_id, self.sample_id, md5_confirm_date)
+            msg = 'MD5 check failed - was checked before on %s' % md5_confirm_date
+
+        emit('Delivery %s sample %s: %s', self.delivery_id, self.sample_id, msg)
```
Comment (tcezard) [inline upload_to_gel/deliver_data_to_gel.py]: In the happy path we will have an error message. were you meant to change the emit variable ?

Comment (mwhamgenomics) [reply upload_to_gel/deliver_data_to_gel.py]: Yup, was missing a set to `info` above - fixed now

File: upload_to_gel/deliver_data_to_gel.py (hunk 1 above)
Comment (mwhamgenomics) [reply upload_to_gel/deliver_data_to_gel.py]: Fixed as per review below

**Pull request merged**